Expensify/App
# fix: input styles

### Details

https://github.com/Expensify/App/issues/5462#issuecomment-927869687

### Fixed Issues

$ https://github.com/Expensify/App/issues/5462

### Tests | QA Steps

Open NewDot on all the below-listed platforms one by one and follow the following procedure.
1. Open Profile Page.
2. Check the Input for FirstName and LastName is proper as per the issue description.
3. Check the Pronoun Dropdown is proper.

### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/136659847-d39cb909-d7d6-441e-9103-a6d9f33c97da.png)

#### Mobile Web


#### iOS


#### Android

![Screenshot 2021-10-09 20:23:03](https://user-images.githubusercontent.com/24370807/136663070-01410b27-9cd5-4603-9b2c-0e4ebfc582c4.png)
